hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -703,7 +703,14 @@ public boolean mkdirs(Path f, FsPermission permission) throws IOException {
 
   @Override
   public long getDefaultBlockSize() {
```

Comment:
We need a fix for this in BasicRootedOzoneFileSystem#getDefaultBlockSize() as well.